h2o-py/h2o/model/model_base.py

```diff
@@ -1713,20 +1714,6 @@ def score_history(self):
         return self.scoring_history()
 
 
-
-
-def _get_matplotlib_pyplot(server):
-    try:
-        # noinspection PyUnresolvedReferences
-        import matplotlib
-        if server: matplotlib.use("Agg")
-        # noinspection PyUnresolvedReferences
-        import matplotlib.pyplot as plt
-        return plt
-    except ImportError:
-        print("`matplotlib` library is required for this function!")
-        return None
-
```

Comment:
wouldn't it make sense to move the other import utility functions below to the same py module?
I know it's out of scope somehow, but it would improve consistency.

Reply:
I agree that it would make sense and it would be nicer to have more consistent package but a simple grep reveals that except for `matplotlib`, there aren't many repeating "try-import" patterns (on this branch):

```
$ cd h2o-py/h2o 
$ rg -U '\s*try:[\w\W]*?(except|finally)' . | perl -pe 's/^.*?:\s*(.*?)(?:\s*#.*)?$/\1/g' | rg '(import|from) ' | sort |uniq -c |sort -nr

      2 import matplotlib.pyplot as plt
      1 raise RuntimeError('Cannot import pandas')
      1 import winkerberos as kerberos
      1 import subprocess
      1 import pandas
      1 import numpy as np
      1 import matplotlib.figure
      1 import matplotlib
      1 import kerberos
      1 import h2o.explanation
      1 import gssapi
      1 import IPython.display
      1 import IPython
      1 from termios import TIOCGWINSZ
      1 from struct import unpack
      1 from sklearn.externals.funcsigs import Parameter, signature
      1 from requests.auth import AuthBase
      1 from mpl_toolkits.mplot3d import Axes3D
      1 from matplotlib import cm
      1 from ipykernel.zmqshell import ZMQInteractiveShell
      1 from inspect import Parameter, signature
      1 from h2o.explanation import explain, explain_row, varimp_heatmap, model_correlation_heatmap, \
      1 from fcntl import ioctl
      1 from distutils.version import LooseVersion
      1 from dis import _unpack_opargs
      1 from ..utils.shared_utils import can_use_numpy, can_use_pandas
```

The fact that those imports are mostly used once makes me think that it's not worth doing in this PR.